gaia/filters.py

```diff
@@ -11,20 +29,57 @@
 
 
 def filter_pandas(df, filters):
```

Comment:
@aashish24 IMO that would be less intuitive, and increase the complexity of running processes.  You'd need to run a nested process to filter the data instead of just defining the filter in the input object.  Plus it doesn't really seem like a spatial process to just add the equivalent of a simple WHERE SQL clause to the dataset input.